text/entities/0256-entities-data-model.md

```diff
@@ -0,0 +1,790 @@
+unnecessary and violates the Minimally Sufficient Id rule.
+
+### Examples of Entities
+
+_This section is non-normative and is present only for the purposes of demonstrating
+the data model._
+
+Here are examples of entities, the typical identifying attributes they
+have and some examples of non-identifying attributes that may be
+associated with the entity.
+
+<table>
+   <tr>
+    <td><strong>Entity</strong>
+    </td>
+    <td><strong>Entity Type</strong>
+    </td>
+    <td><strong>Identifying Attributes</strong>
+    </td>
+    <td><strong>Non-identifying Attributes</strong>
+    </td>
+   </tr>
+   <tr>
+    <td>Service
+    </td>
+    <td>"service"
+    </td>
+    <td>service.name (required)
+<p>
+service.instance.id
+<p>
+service.namespace
+    </td>
+    <td>service.version
+    </td>
+   </tr>
+   <tr>
+    <td>Host
+    </td>
+    <td>"host"
+    </td>
+    <td>host.id
+    </td>
+    <td>host.name
+<p>
+host.type
+<p>
+host.image.id
+<p>
+host.image.name
+    </td>
+   </tr>
+   <tr>
+    <td>K8s Pod
+    </td>
+    <td>"k8s.pod"
+    </td>
+    <td>k8s.pod.uid (required)
+<p>
+k8s.cluster.name
+    </td>
+    <td>Any pod labels
+    </td>
+   </tr>
+   <tr>
+    <td>K8s Pod Container
+    </td>
+    <td>"container"
+    </td>
+    <td>k8s.pod.uid (required)
+<p>
+k8s.cluster.name
+<p>
+container.name
+    </td>
+    <td>Any container labels
+    </td>
+   </tr>
+</table>
+
+See more examples showing nuances of Id field composition in the
+[Entity Identification](#entity-identification) section.
+
+## Entity Events
+
+Information about Entities can be produced and communicated using 2
+types of entity events: EntityState and EntityDelete.
+
+### EntityState Event
+
+The EntityState event stores information about the _state_ of the entity
+at a particular moment of time. The data model of the EntityState event
+is the same as the entity data model with some extra fields:
+
+<table>
+   <tr>
+    <td><strong>Field</strong>
+    </td>
+    <td><strong>Type</strong>
+    </td>
+    <td><strong>Description</strong>
+    </td>
+   </tr>
+   <tr>
+    <td>Timestamp
+    </td>
+    <td>nanoseconds
+    </td>
+    <td>The time since when the entity state is described by this event.
+The time is measured by the origin clock. The field is required.
+    </td>
+   </tr>
+   <tr>
+    <td>Interval
+    </td>
+    <td>milliseconds
+    </td>
+    <td>Defines the reporting period, i.e. how frequently the
+information about this entity is reported via EntityState events even if
+the entity does not change. The next expected EntityEvent for this
+entity is expected at (Timestamp+Interval) time. Can be used by
+receivers to infer that a no longer reported entity is gone, even if the
+EntityDelete event was not observed. Optional, if missing the interval
+is unknown.
+    </td>
+   </tr>
+   <tr>
+    <td>Type
+    </td>
+    <td>
+    </td>
+    <td>See data model.
+
+</td>
+   </tr>
+   <tr>
+    <td>Id
```

Comment:
Is the entity state used to create entities, and does the uniqueness of this Id cause the creation when the entity is not already found?

Reply:
Yes and yes.